src/server.c

```diff
@@ -3292,6 +3295,17 @@ void resetCommandTableStats(void) {
 
 }
 
+void resetErrorTableStats(void) {
+    raxIterator ri;
+    raxStart(&ri,server.errors);
+    raxSeek(&ri,"^",NULL,0);
+    while(raxNext(&ri)) {
+        struct redisError *err = ri.data;
+        if(err) err->count=0;
+    }
+    raxStop(&ri);
```

Comment:
i think you just need to discard the rax and create a fresh one.
this way the INFO response won't even show 0 values for errors that no longer happen.

Reply:
@oranagra given that we check on the `info errorstats` string building if the error is 0 and don't print it (`if(e->count)`) shouldn't we keep the rax to avoid uncessary destroy and construction?

Sample output validating the above:
```
127.0.0.1:6379> auth 1
(error) ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?
127.0.0.1:6379> info errorstats
# Errorstats
errorstat_auth:count=1
127.0.0.1:6379> config resetstat
OK
127.0.0.1:6379> info errorstats
# Errorstats
```

Reply:
yeah, you're right about the output. sorry,
well, many places in redis just discard the rax or dict and create an empty one in similar situation, i suppose it avoids keeping a lot of elements that are relics past traffic (in this case errors might no longer be seen again).
but i suppose there are not that may error types, so i guess we can go either way.